packages/sdk/src/doc/common.ts

```diff
@@ -13,4 +13,5 @@ export interface IRpcRequest {
     command: string;
 }
 
-export type IRpcResponse = {};
+// biome-ignore lint/suspicious/noEmptyInterface: <explanation>
+export interface IRpcResponse {}
```

Comment:
Regarding this `biome-ignore` comment, why is this an empty interface? I don't have anything against empty interfaces, but we might want to explain why for developer clarity 😋